packages/cloud-cognitive/src/components/ButtonSetWithOverflow/ButtonSetWithOverflow.js

```diff
@@ -206,24 +226,12 @@ ButtonSetWithOverflow.propTypes = {
    * onResize reports maxSize on resize
    */
   onWidthChange: PropTypes.func,
-  /**
-   * pageActionsLabel - used when button set is shown as combo button
-   */
-  pageActionsLabel: PropTypes.node,
   /**
    * align buttons to right of available space
    */
   rightAlign: PropTypes.bool,
-  /**
-   * Specify the size of the button, from a list of available sizes.
-   * For `default` buttons, this prop can remain unspecified.
-   */
-  size: Button.propTypes.size,
 };
 
-ButtonSetWithOverflow.defaultProps = {
-  pageActionsLabel: 'Page actions',
-  size: 'field',
-};
+ButtonSetWithOverflow.defaultProps = {};
```

Comment:
We could remove this line entirely now since there are no longer any default props.

Reply:
Good plan I'll include it in part 2.